lesson_02/test_script.py

```diff
@@ -26,10 +23,10 @@ def run_job1():
     assert resp.status_code == 201
     print("job1 completed!")
 
-def run_job2():
+def run_job2(port=8082, host='localhost'):
```

Comment:
not need add default value
Suggested change
def run_job1(port: int, host: str):